lib/core/src/lib/auth/oidc/auth-config.service.ts

```diff
@@ -98,4 +94,11 @@ export class AuthConfigService {
     private getLocationOrigin() {
         return window.location.origin;
     }
+
+    private generatePostLogoutUri(hostUri: string = '', redirectUriLogout: string = ''): string {
+        const hostUriWithoutSlash = hostUri.endsWith('/') ? hostUri.substring(0, hostUri.length - 1) : hostUri;
+        const redirectUriLogoutWithoutSlash = redirectUriLogout.startsWith('/') ? redirectUriLogout.substring(1) : redirectUriLogout;
```

Comment:
this is not fail proof for "/" values ?

Reply:
you mean for `redirectUriLogout` ?

[here is ](https://github.com/Alfresco/alfresco-ng2-components/pull/10122/files#diff-9da78e2933647dc1f91d6381efdce6ec98e18de342510c46011d9ccef699f811R197) test for `redirectUriLogout` and `/` value